### DISCUSSION TITLE
Feedback on `/docs/slider/`

### DISCUSSION THREAD

**Discussion by DanAbney** [iTwin/iTwinUI#1047] [Feedback]
This seems to be missing the properties. Looking for a way to make this slider "incremental" (instead of free sliding).

**Comment by mayank99** [comment]
This is a known issue, there are other components missing props too (for example, FileUpload). For now you can use the storybook while we work on adding examples.

To answer your question about slider, there is no concept of "free sliding". I believe you are looking for one of these:
- the `step` prop will let you configure the value of each increment. ([example](https://itwin.github.io/iTwinUI/react/?path=/story/input-slider--decimal-increment))
- the `max` prop. if you use a small value, you'll see the increments more clearly. ([example](https://itwin.github.io/iTwinUI/react/?path=/story/input-slider--decimal-increment&args=max:10;step:2;values[0]:5))
